### PR TITLE
Update upgrades.md

### PR DESCRIPTION
**Pull request by martinmosegaard**
Changed "Node:" to "Note:" because it was a note :)
Also a few minor changes for clarification.

> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7333)

### CODE REVIEW

File: docs/getting-started-guides/ubuntu/upgrades.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ to ensure that the cluster upgrade has successfully completed.
 ### Upgrade Flannel
 
 Upgrading flannel can be done at any time, it is independent of Kubernetes upgrades.
-Be advised that networking is interrupted during the upgrade. You can initiate a flannel upgrade:
+Be advised that networking is interrupted during the upgrade:
```
Comment (tengqm) [inline docs/getting-started-guides/ubuntu/upgrades.md]: The old text reads better to me.

Comment (martinmosegaard) [reply docs/getting-started-guides/ubuntu/upgrades.md]: No problem, I will revert the change. I forgot to motivate the change: In the paragraph below about the easyrsa upgrade, there is no such introductory sentence.

**Pull request merged**